book/website/communication/podcasts/podcasts-personal-stories.md

```diff
@@ -0,0 +1,92 @@
+(cm-podcasts-personal-stories)=
+# Personal stories - Podcasts
+
+## Zoe Ingram and Emma Harris: Orion Open Science Podcast
+
+The ORION Open Science podcast is structured around a series of interviews with interesting guests about a variety of topics related to Open Science.
+
+It is also co-hosted and co-produced by **Dr. Luiza Bengtsson**.
+
+If you want to find out more information about the podcast then look at their [website](https://www.orion-openscience.eu/publications/training-materials/201902/podcasts) and the [podcast on PodBean](https://orionopenscience.podbean.com/).
+
+We would like to thank **Zoe Ingram** and **Emma Harris** for answering our interview questions about their very successful and enjoyable blog series.
```

Comment:
You have boldened contributors here but you haven't done that in other sections, such as the interview with Emma. It would probably be good to pick a style and stick to it.